README.md

```diff
@@ -45,6 +45,7 @@ This library currently supports the following cryptocurrencies and address forma
  - EOS
  - ETC (checksummed-hex)
  - ETH (checksummed-hex)
+ - FIL ([bls](https://spec.filecoin.io/#section-glossary.address))
```

Comment:
BLS is the signature scheme, not the address encoding; the address encoding is base32 + blake2b checksum